manifests/mets.py

```diff
@@ -469,12 +467,14 @@ def main(data, document_id, source, host, cookie=None):
 		while not_paged:
 		  try:
 		     metadata_url =  metadata_url_base + cursormark_val
-		     response = webclient.get(metadata_url, cookie)
-		  except urllib.request.HTTPError as err:
+		     cookies = {'hulaccess': cookie}
```

Comment:
Oh yeah, _definitely_ should see perf benefits from keeping a single requests.Session with the cookies in it and reusing it.